list/list.go

```diff
@@ -61,7 +63,7 @@ func (l *List) Prev() {
 }
 
 // Search allows the list to be filtered by a given term. The list must
-// implement the searcher method for that.
+// implement the searcher function signature for this functionality to work.
```

Comment:
really, it must have a value set for its Searcher field.

Reply:
I'll make some precision, this was a bit hard to word properly.